graphdatascience/query_runner/arrow_query_runner.py

```diff
@@ -377,10 +377,17 @@ def __init__(self, factory: AuthFactory, *args: Any, **kwargs: Any) -> None:
         self._factory = factory
 
     def received_headers(self, headers: Dict[str, Any]) -> None:
-        auth_header: str = headers.get("Authorization", None)
+        auth_header = headers.get("authorization", None)
```

Comment:
for the AuraDbArrowQueryRunner we did use the lower case already

Reply:
indeed